### PR TITLE
fix(dapp): use env vars instead of dotenv

### PR DESCRIPTION
**Pull request by valera-rozuvan**

### CODE REVIEW

File: .github/workflows/ci-production.yml [hunk 1]
```diff
@@ -30,34 +30,20 @@ jobs:
...
+          REACT_APP_LOG_LEVEL: debug
+          REACT_APP_MODE: production
+          REACT_APP_GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
+          REACT_APP_FILE_WEB3STORAGE_KEY: ${{ secrets.FILE_WEB3STORAGE_KEY }}
+          REACT_APP_NETWORK_ID: ${{ secrets.NETWORK_ID }}
+          REACT_APP_NETWORK_PROVIDER: ${{ secrets.NETWORK_PROVIDER }}
+          REACT_APP_CONTRACT_ADDRESS: ${{ secrets.CONTRACT_ADDRESS }}
+          REACT_APP_SEARCH_FORM_DISABLED: ${{ secrets.SEARCH_FORM_DISABLED }}
         run: |
           rm -rf ./docs
```
Comment (mfw78) [inline .github/workflows/ci-production.yml:48]: suggest insert `export` here for absolute debugging.

**Review by mfw78 — APPROVED** [review]
Minor comment on debugging using `export`

**Review by kostysh — APPROVED** [review]
LGTM

File: .github/workflows/ci-production.yml (hunk 1 above)
Comment (valera-rozuvan) [reply .github/workflows/ci-production.yml:48]: Why? We definitely know that env variables are being applied ...

Just took a look of how you do it here:

https://github.com/windingtree/lpms-server/blob/main/.github/workflows/release.yml

```
    - name: Release
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
      run: npx semantic-release
```

You don't use `export` @mfw78 .

**Pull request merged**